homeassistant/components/apple_tv/media_player.py

```diff
@@ -213,7 +202,7 @@ def media_title(self):
             title = self._playing.title
             return title if title else "No title"
 
-        return f"Establishing a connection to {self._name}..."
+        return self._manager.message
```

Comment:
It seems we're using media title to display what the tv is doing, connecting etc. That's not intended.

Reply:
I am and I know, but there's no other way to provide any feedback to the user?

Reply:
There's currently no way to do that directly in the GUI. This should probably be a new websocket command for the media player integration, but that's out of scope for this PR, and it would require an architecture discussion.

We should not use the entity state properties for this information.

Reply:
Ok, too bad 😕 I do however need some way to signal back to the user what is happening because I receive a lot of issue reports where this has helped. Maybe I can use some already defined state, like unknown and return error via state attributes instead for the time being? Then it's at least possible to see what happens without having to enable debug logs.

Reply:
If there's an error we should log an error, warning etc. We shouldn't use the state machine for this.

Reply:
We can set state to `None` which will be converted to unknown state if we don't know the state. If we know that the connection to the device is not working we should set `available` to False.

Reply:
This is not necessarily an error, it might just be a short hiccup because the scanning failed. It's not 100% reliable, so it might take a few attempts. It also reflects the initial connection attempt.

Reply:
One problem with setting `available` to `False` is that you can't interact with the entity at all, so you can't turn it off easily from the UI, right? But I do see that there's no solution here and I will remove all my status code.